### ISSUE TITLE
param_list_to_url_string function in util.py function is not associated with any class but uses self parameter

### ISSUE THREAD

**Issue by njanakiev** [geopython/OWSLib#662]
When running `getCoverage` in `owslib=0.19.1` for:

```python
geoserver_url = "..."
layer_name = "..."
lat_subset  = ('Lat',   40.47739,  40.916178)
long_subset = ('Long', -74.25909, -73.700181)

wcs = WebCoverageService(geoserver_url, version='2.0.1')

output = wcs.getCoverage(identifier=[layer_name],
    format='image/tiff', subsets=[long_subset, lat_subset])
```
I get the error: `TypeError: param_list_to_url_string() missing 1 required positional argument: 'param_name'` This snippet worked fine for `owslib=0.19.0`. 

I can see that in `utils.py`, the function `param_list_to_url_string(self, param_list, param_name)` uses `self` but is not associated to any class.

**Comment by cehbrecht** [comment]
@njanakiev this looks like a bug. Would you like to make a PR?

**Comment by Dukestep** [comment]
Sorry, I introduced this... I submitted a PR that rectifies this error (#663).

**Comment by cehbrecht** [comment]
fixed by PR #663.
